hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieMetadataWriteUtils.java

```diff
@@ -147,6 +148,10 @@ public static HoodieWriteConfig createMetadataWriteConfig(
     final Properties properties = new Properties();
     properties.put(HoodieTableConfig.RECORDKEY_FIELDS.key(), RECORD_KEY_FIELD_NAME);
     properties.put("hoodie.datasource.write.recordkey.field", RECORD_KEY_FIELD_NAME);
+    if (nonEmpty(writeConfig.getMetricReporterMetricsNamePrefix())) {
+      properties.put(HoodieMetricsConfig.METRICS_REPORTER_PREFIX.key(),
+          writeConfig.getMetricReporterMetricsNamePrefix() + "_metadata");
+    }
```

Comment:
Where does the `_metadata` constant string come from?

Reply:
guess, we are deriving the metrics name prefix from data table's write config. and hence suffixing "_metadata" to it.

Reply:
Yes you are right. I have made a change to use `METADATA_TABLE_NAME_SUFFIX` instead here.